app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.kt

```diff
@@ -198,15 +197,15 @@ class FahrplanFragment : Fragment(), SessionViewEventsHandler {
         activity.invalidateOptionsMenu()
         val intent = activity.intent
         // TODO Consume session alarm in MainActivity and let it orchestrate its fragments
-        val sessionId = intent.getStringExtra(BUNDLE_KEY_SESSION_ALARM_SESSION_ID)
+        val sessionId = intent.getStringExtra(BundleKeys.BUNDLE_KEY_SESSION_ALARM_SESSION_ID)
```

Comment:
Please use static imports when constants are prefixed with `BUNDLE_KEY` here and in other files.

Reply:
Please check again. I was only referring to the prefixed constants.

Reply:
Sorry, must have overlooked the other Flies 🤦 Should be done now

Reply:
Not really. But instead of fixing only the imports I suggest we consolidate the name of the constants once and forever. It spreads too much confusion. Please use the **same naming schema** for all bundle keys. You decide:

- Option 1: No upper case `BUNDLE_KEY_` prefix. Then don't use static imports but always use `BundleKeys.SOME_KEY` when you reference a constant.
- Option 2: Always use upper case `BUNDLE_KEY_` prefix. Then always use static imports because the name of the constant describes it perfectly.

Reply:
> Not really. But instead of fixing only the imports I suggest we consolidate the name of the constants once and forever. It spreads too much confusion. Please use the **same naming schema** for all bundle keys. You decide:
> 
>     * Option 1: No upper case `BUNDLE_KEY_` prefix. Then don't use static imports but always use `BundleKeys.SOME_KEY` when you reference a constant.
> 
>     * Option 2: Always use upper case `BUNDLE_KEY_` prefix. Then always use static imports because the name of the constant describes it perfectly.

I must have misread your message, I thought you meant to remove every `BundleKeys.` Prefix, sorry that was my fault. I'll go with Option 1, as it then is easier to see where the Key comes from.

Reply:
You forgot `AlarmServiceTest`.

Reply:
Jupp saw that I also somehow changed some other stuff.